docs/en/getting-started/install.md

```diff
@@ -406,3 +347,65 @@ SELECT 1
 **Congratulations, the system works!**
 
 To continue experimenting, you can download one of the test data sets or go through [tutorial](/docs/en/tutorial.md).
+
+## Self-Managed Requirements
+
+### CPU Architecture
+
+ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture.
+
+Official pre-built binaries are typically compiled for x86_64 and leverage SSE 4.2 instruction set, so unless otherwise stated usage of CPU that supports it becomes an additional system requirement. Here’s the command to check if current CPU has support for SSE 4.2:
```

Comment:
I'd consider the SSE level on x86 a detail which users should not have to worry about. Besides that, SSE4.2 was released in 2006 and there exist only very very few systems without it these days.

More importantly, with #44366, I have changed the "universal.sh" script (that should be the default recommendation) such that it now fetches the right SSE build on x86 independently of whether the system supports SSE4.2 or not.

@DanRoscigno Suggest to
- rewrite l. 357 as "Official binaries are available for x86-64 and AArch64.",
- remove l. 358-363,
- Instead add a general sentence à la "It is also possible to build ClickHouse from source, see [...] for details."

Reply:
Thanks @rschu1ze , based on the above, can I remove the second sentence of line #260 also:

> You can compile packages and install them or use programs without installing packages. Also by building manually you can disable SSE 4.2 requirement or build for AArch64 CPUs.